sections/rich-text.liquid

```diff
@@ -165,9 +165,9 @@
       "limit": 3,
       "settings": [
         {
-          "type": "richtext",
+          "type": "inline_richtext",
```

Comment:
@kjellr for the same reason Ludo shared above I would be suggesting to go for `richtext` for image banner to keeping consistency among similar element.

Reply:
@melissaperreault and I chatted offline, and we want to clarify the situation here. 

We _are_ ok with losing inline formatting . So for example, if the merchant had `<br />` tags in there, and we're losing them upon upgrade, that's acceptable if we give them a heads up about it. 

But @ludoboludo, it sounds like [from your description up here](https://github.com/Shopify/dawn/pull/2085#discussion_r1040187842), we're losing the merchant's text entirely, is that correct? 

If that's the case, that's not ok. We don't want to have content loss on upgrade. If this is the case (and we can't fix it with something like [the transformation suggestion](https://github.com/Shopify/shopify/blob/main/components/online_store/app/models/theme/setting_updater.rb#L9)), then we should keep this field as is for now.

Reply:
Ok, thanks for the clarification. @melissaperreault and I don't want to lose any content, but we also don't want to hold up this PR. 

So for now, I've removed this `richtext` > `inline_richtext` change from the PR (https://github.com/Shopify/dawn/pull/2085/commits/610e6c42027ef3ef7c558c224ede7777a451e3f0). I did leave the `inline-richtext` class on the Rich Text Heading field though, so that the hover color for that matches the hover state for other headings (https://github.com/Shopify/dawn/pull/2085/commits/2659ebade4c51e169164bcc4187785dc1d560dc4). 

Let's explore this `richtext` > `inline_richtext` question separately in a followup PR. I'll mark [these todo decisions as completed for now](https://github.com/Shopify/dawn/pull/2085#issuecomment-1342851198), and also note this outcome in the decision log above. 

In the meantime, I'd welcome one more review to make sure nothing else seems off about this PR! Thanks. 🙌